proto/money/folio/requested_reservation_posting.proto

```diff
@@ -0,0 +1,14 @@
+syntax = "proto3";
+
+package holms.types.money.folio;
+option csharp_namespace = "HOLMS.Types.Money.Folio";
+option go_package = "money/folio";
+
+import "booking/indicators/reservation_indicator.proto";
+import "money/folio/requested_charge_credit_posting.proto";
+
+message RequestedReservationPosting {
+	.holms.types.booking.indicators.ReservationIndicator reservation = 1;
```

Comment:
this is new. we used to send a list of `RequestedChargeCreditPosting`s, now we send this.